person.rb

```diff
@@ -1,18 +1,31 @@
 require 'securerandom'
-
-class Person
+require_relative '/nameable'
+require_relative '/capitalize_decorator'
+require_relative '/trimmer_decorator'
+require_relative '/rental'
```

Comment:
- [ ] Please remove the `/` from your import because if your file is on the same path that you want to import in another file you just need to require 
```ruby
   require_relative 'JUST_THE_NAME' # No path NO file extension
```

backslashes are only required if you are using the normal way of importing. I am getting an error related to it that's why It would be nice if your remove all those `forward backslashes`.

```suggestion
require_relative 'nameable'
require_relative 'capitalize_decorator'
require_relative 'trimmer_decorator'
require_relative 'rental'
```

### Error

![Screenshot from 2023-10-05 07-51-13](https://github.com/mohamedgamaleldin999999/School-library/assets/66476329/88692144-7810-455d-86ac-08e41c66eff9)